api/v1alpha4/conditions_consts.go

```diff
@@ -63,3 +63,25 @@ const (
 	// ScaleSetProvisionFailedReason used for failures during scale set provisioning.
 	ScaleSetProvisionFailedReason = "ScaleSetProvisionFailed"
 )
+
+// AzureMachinePool Conditions and Reasons
+const (
+	// PoolRunningCondition reports on current status of the Azure scale set.
+	PoolRunningCondition clusterv1.ConditionType = "PoolRunning"
```

Comment:
what's the difference between `ScaleSetRunningCondition` above and `PoolRunningCondition`? They have the same description

Reply:
I think this was a merge resolution issue. Fixing.